main_routers/agent_router.py

```
@@ -24,6 +25,52 @@
+        return "ko"
+    if lowered.startswith("ru"):
+        return "ru"
+    return "zh-CN"
+
+
+def _load_openclaw_guide_markdown(lang: str | None = None) -> str:
+    resolved_lang = _normalize_openclaw_guide_lang(lang)
+    candidate = _OPENCLAW_GUIDE_LANG_FILES.get(resolved_lang, _OPENCLAW_GUIDE_PATH)
+    try:
+        return candidate.read_text(encoding="utf-8")
+    except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)